maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/booterclient/DefaultForkConfiguration.java

```diff
@@ -251,9 +251,9 @@ private String interpolateArgLineWithPropertyExpressions()
     }
 
     @Nonnull
-    private static String stripWhitespace( @Nonnull String argLine )
+    static String stripWhitespace( @Nonnull String argLine )
```

Comment:
With this access change I expected to see plenty of new UTs - but there is none committed here.

Reply:
Ah, well, I thought about doing that, but eventually refrained from it. I'll revert the visibility change.